src/main/java/com/onarandombox/MultiverseCore/utils/PlayerFinder.java

```diff
@@ -0,0 +1,176 @@
+package com.onarandombox.MultiverseCore.utils;
+
+import com.dumptruckman.minecraft.util.Logging;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * Helper class to get {@link Player} from name, UUID or Selectors.
+ */
+public class PlayerFinder {
+
+    private static final Pattern UUID_REGEX = Pattern.compile("[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[34][0-9a-fA-F]{3}-[89ab][0-9a-fA-F]{3}-[0-9a-fA-F]{12}");
```

Comment:
I think this regex pattern is too strict. What comes to mind is Floodgate UUIDs, https://github.com/GeyserMC/Geyser/wiki/FAQ#how-do-i-find-a-players-uuid-without-them-joining-when-using-floodgate.

Reply:
Hmm yea maybe this will do:
```
[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}
```